akka-serialization-jackson/src/main/scala/akka/serialization/jackson/JacksonSerializer.scala

```diff
@@ -86,6 +81,50 @@ import akka.util.Helpers.toRootLowerCase
     (bytes(0) == GZIPInputStream.GZIP_MAGIC.toByte) &&
     (bytes(1) == (GZIPInputStream.GZIP_MAGIC >> 8).toByte)
   }
+
+  case class LZ4Meta(offset: Int, length: Int) {
+    import LZ4Meta._
+
+    def putInto(buffer: ByteBuffer): Unit = {
+      buffer.put(LZ4_MAGIC.toByte)
+      buffer.put((LZ4_MAGIC >> 8).toByte)
+      buffer.putInt(length)
+    }
+
+  }
+
+  object LZ4Meta {
+    val LZ4_MAGIC = 0x6df6 // The last 2 bytes of `printf akka | sha512sum`
```

Comment:
could we use same magic as in the lz4 framing (even though we don't use framing)

Reply:
I don't think this is a good idea. The frame format reference shows that after the magic number there should be some metadata information. Standard LZ4 frame decompressor shouldn't be able to decompress our compressed data anyway, because we don't have the metadata included in our compressed data. To avoid the confusion, I think we'd better use a new magic number,

Reply:
I agree

Reply:
could you just double check that this `0x6df6` isn't accidentally conflicting with typical json and cbor start bytes

Reply:
I have added extra two bytes for the magic number. Both the current value `0x87d96df6` and `0x6df6` shouldn't be typical json or cbor start bytes. `87` is not within range ascii. In cbor it represents a array of length 7. `d9` is not assigned.